release-notes/opensearch-release-notes-2.15.0.md

```diff
@@ -0,0 +1,740 @@
+
+
+### Opensearch Neural Search
+
+
+* Pass empty doc collector instead of top docs collector to improve hybrid query latencies by 20% ([#731](https://github.com/opensearch-project/neural-search/pull/731))
+* Optimize parameter parsing in text chunking processor ([#733](https://github.com/opensearch-project/neural-search/pull/733))
+* Use lazy initialization for priority queue of hits and scores to improve latencies by 20% ([#746](https://github.com/opensearch-project/neural-search/pull/746))
+* Optimize max score calculation in the Query Phase of the Hybrid Search ([765](https://github.com/opensearch-project/neural-search/pull/765))
+* Implement parallel execution of sub-queries for hybrid search ([#749](https://github.com/opensearch-project/neural-search/pull/749))
+
+
+### Opensearch Security
+
+
+* Replace BouncyCastle's OpenBSDBCrypt use with password4j for password hashing and verification ([#4428](https://github.com/opensearch-project/security/pull/4428))
+* Adds validation for the action groups type key ([#4411](https://github.com/opensearch-project/security/pull/4411))
+* Made sensitive header log statement more clear ([#4372](https://github.com/opensearch-project/security/pull/4372))
+* Refactor ActionGroup REST API test and partial fix #4166 ([#4371](https://github.com/opensearch-project/security/pull/4371))
+* Support multiple audience for jwt authentication ([#4363](https://github.com/opensearch-project/security/pull/4363))
+* Configure masking algorithm default ([#4345](https://github.com/opensearch-project/security/pull/4345))
+* Remove static metaFields list and get version-specific values from core ([#4412](https://github.com/opensearch-project/security/pull/4412))
+
+
+### Opensearch Security Dashboards Plugin
+
+
+* Remove tenant tab when disabled via yaml ([#1960](https://github.com/opensearch-project/security-dashboards-plugin/pull/1960))
+* Always show security screen and shows error page when trying to access forbidden data-source ([#1964](https://github.com/opensearch-project/security-dashboards-plugin/pull/1964))
+* Provide ability to view password ([#1980](https://github.com/opensearch-project/security-dashboards-plugin/pull/1980))
+* Make login screen input feels consistent ([#1993](https://github.com/opensearch-project/security-dashboards-plugin/pull/1993))
+
+
+### Opensearch k-NN
+
+
+* Add KnnCircuitBreakerException and modify exception message ([#1688](https://github.com/opensearch-project/k-NN/pull/1688))
+* Add stats for radial search ([#1684](https://github.com/opensearch-project/k-NN/pull/1684))
+* Support script score when doc value is disabled and fix misusing DISI ([#1696](https://github.com/opensearch-project/k-NN/pull/1696))
+* Add validation for pq m parameter before training starts ([#1713](https://github.com/opensearch-project/k-NN/pull/1713))
+* Block delete model requests if an index uses the model ([#1722](https://github.com/opensearch-project/k-NN/pull/1722))
+
+
+### OpenSearch SQL
```

Comment:
```suggestion
### SQL
```

Reply:
Seems like before 2.10, we were using `OpenSearch SQL` and `SQL` after. I feel like `OpenSearch SQL` sounds more standardized and we should update all previous ones. 
https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-2.8.0.md#opensearch-sql
https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-2.7.0.md#opensearch-sql

Reply:
Yes @gaiksaya that is why I switch back to OpenSearch SQL to be consistent.